projects/WeTek_Core/initramfs/platform_init

```diff
@@ -1,20 +1,20 @@
 #!/bin/sh
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
-#      Copyright (C) 2014 Alex Deryskyba (alex@codesnake.com)
```

Comment:
@CvH 
> The problem is nobody knows who has written ...

does not matter.

I am the author of near half of LE. I never added myself as an author. nobody cares. but no need to remove people from license headers. and adding yourself as the only author in a file, where you only chanced a copyright header is obviously not the right thing to do. when in doubt - git log is your best friend. and dont touch names/emails. replacing OpenELEC with LibreELEC is perfectly fine as OE copyright does not apply to this git repository.